server/bundler/utils.js

```diff
@@ -14,6 +14,9 @@ function getAssets( stats ) {
 }
 
 function pathToRegExp( path ) {
+	if ( path === '/' ) {
+		return path;
+	}
 	return new RegExp( '^' + path + '(/.*)?$' );
```

Comment:
> that means changing all existing route definitions in wordpress-com and making sure they are kept in sync between the section definition and the route handling.

also isn't this part of the problem that we're trying to fix? our routes _aren't_ all explicit and so we can let through routes that don't exist? no objection here to shipping and _improving_ the system but I think yeah we should consider addressing the real problem which is that we're loose on paths and routing